Yoast/YoastSEO.js
# Move files to fit a monorepo structure

## Summary

This PR can be summarized in the following changelog entry:

* Move all files in the repository to fit a monorepo structure.

## Relevant technical choices:

* I decided to let Travis just `cd` into the `packages/yoastseo` directory so it all simply works as before.

## Test instructions

This PR can be tested by following these steps:

* Make sure Travis still works as expected
* Make sure you can still link YoastSEO.js to a Yoast SEO installation and you can still build all the necessary files for the content analysis.
